pandas/tests/arrays/interval/test_interval.py

```diff
@@ -376,3 +376,18 @@ def test_arrow_table_roundtrip_without_metadata(breaks):
     result = table.to_pandas()
     assert isinstance(result["a"].dtype, pd.IntervalDtype)
     tm.assert_frame_equal(result, df)
+
+
+def test_tz_unique():
+    # GH 46128
+    dti1 = date_range("2016-01-01", periods=3)
+    ii1 = IntervalIndex.from_breaks(dti1)
+    ser1 = pd.Series(ii1)
```

Comment:
can you either put this in the series tests or test the array method directly.  ideally do a tm.assert_foo_equal below, should be easy bc .unique matches self in these cases

Reply:
OK, moved to series tests in next commit.